ledger/sandbox-on-x/src/test/it/scala/com/daml/ledger/platform/sandbox/auth/OngoingStreamAuthIT.scala

```diff
@@ -101,7 +101,7 @@ final class OngoingStreamAuthIT
           new AssertionError("Timed-out waiting while waiting for stream to abort")
         )
       }
-      _ = new Timer(true).schedule(timerTask, 100)
+      _ = new Timer(true).schedule(timerTask, 1000)
```

Comment:
Is using the same timeout for observing stream termination and the cache expiry intentional? Could you clarify what happens if the termination doesn't fits in this 1 second period? Would it race with the cache expiry timeout in some way?

Reply:
There's a thread sleep just before this timer and after granting a new right:
```
_ = Thread.sleep(UserManagementCacheExpiryInSeconds.toLong * 1000)
```
This timeout is on-top of the above one. Does it clarify you concerns?

Reply:
Changed the code, please take another look

Reply:
🤦  You're obviously right - it was 2 seconds delay from the start.
It looks clearer now though. Thank you!